src/legacy/core_plugins/tests_bundle/tests_entry_template.js

```diff
@@ -83,6 +83,9 @@ new CoreSystem({
         enabled: true,
         enableExternalUrls: true
       },
+      interpreterConfig: {
+        enableInVisualize: true
```

Comment:
To make sure I understand: This means, moving forward, all tests will be run with pipeline enabled, correct?

Reply:
yes

Reply:
okay, so do we need to include `enabled: true` here as well? (i see that it is there for vega)